hipchat_room_message

```diff
@@ -75,19 +79,40 @@ if [[ -z $TOKEN ]] || [[ -z $ROOM_ID ]] || [[ -z $FROM ]]; then
   exit 1
 fi
 
-# nagios levels
+# Override color based on level / status
 if [ ! -z "$LEVEL" ]; then
-  if [[ $LEVEL == 'CRITICAL' ]] || [[ $LEVEL == 'critical' ]]; then
+  LEVEL=$(echo $LEVEL | tr '[:lower:]' '[:upper:]')
+
+  if [ ! -z "$STATUS" ]; then
+    # Map Level to zabbix trigger severity
+    STATUS=$(echo $STATUS | tr '[:lower:]' '[:upper:]')
+    if [[ "$STATUS" == 'OK' ]]; then
+      COLOR="green";
+    elif [[ $LEVEL == 'DISASTER' ]]; then
```

Comment:
should these be $STATUS instead of $LEVEL?

Reply:
I don't have a running Zabbix instance to test against anymore to verify this, however I believe the PR is correct as it exists. The Zabbix Status is either "OK" or "PROBLEM", and if it is "PROBLEM" then Level is used to map the severity.